app/helpers/details_table_helper.rb

```diff
@@ -1,14 +1,42 @@
 module DetailsTableHelper
-  def details_html(attribute)
+  def details_html(attribute, resource = nil)
+    resource_class = resource.class.name
+
     if attribute[:format] == :bullet && attribute[:value].length > 1
       list = attribute[:value].map { |value| "<li>#{value}</li>" }.join
       simple_format(list, { class: "govuk-list govuk-list--bullet" }, wrapper_tag: "ul")
     else
-      return simple_format(attribute[:value].first.to_s, { class: "govuk-body" }, wrapper_tag: "p") if attribute[:value].is_a?(Array)
-
-      value = attribute[:value].presence || "<span class=\"app-!-colour-muted\">You didn’t answer this question</span>".html_safe
+      return simple_format(attribute[:value].first.to_s, { class: "govuk-body" }, wrapper_tag: "p") if attribute[:value].is_a?(Array) && attribute[:value].any?
```

Comment:
Added the `attribute[:value].any?` to ensure that the value is not only an Array but also contains at least one element, preventing the code from attempting to access the first element of an empty array which would cause an error. It'd instead return "No answer provided" when there are no values in the list.